runtime/doc/api.txt

```diff
@@ -700,6 +700,21 @@ nvim_chan_send({chan}, {data})                              *nvim_chan_send()*
       • {chan}  id of the channel
       • {data}  data to write. 8-bit clean: can contain NUL bytes.
 
+nvim_complete_set({index}, {*opts})                      *nvim_complete_set()*
+    Set info for the completion candidate index. if the info was shown in a
+    window, then the window and buffer ids are returned for further
+    customization. If the text was not shown, an empty dict is returned.
```

Comment:
@glepnir when would it *not* show the info in a window? Not clear from the tests.

Reply:
when index not match the list_idx in compl_match double linked list, the result is empty mean failed.